store/tikv/2pc.go

```diff
@@ -497,6 +497,7 @@ func (c *twoPhaseCommitter) buildPrewriteRequest(batch batchKeys, txnSize uint64
 		IsPessimisticLock: isPessimisticLock,
 		ForUpdateTs:       c.forUpdateTS,
 		TxnSize:           txnSize,
+		MinCommitTs:       c.startTS + 1,
```

Comment:
should be `c.forUpdateTS + 1` if its not 0.

Reply:
Done